### ISSUE TITLE
[Windows] Installing a game in "Program Files" fails because it has space in it 

### ISSUE THREAD

**Issue by Zeswen** [Heroic-Games-Launcher/HeroicGamesLauncher#711]
**Describe the bug**
Installing a game in the `Program Files` folder creates a new folder named `Program` (which Microsoft claims it could make some applications fail) in the drive's directory.

**To Reproduce**

1. Go to any game
2. Click on install
3. Select a new folder in `Program Files`
4. A folder named `Program` will appear in the drive

**Expected behavior**
The installation should work properly and the subdirectory as well.

**Desktop (please complete the following information):**

- OS: Windows 11 
- Version 22483
- Game: Kingdom Hearts 3
- Wine/Proton version: None
- Hardware specs (RAM, CPU, GPU): 32GB 3400 MHz, Ryzen 9 5900X, Palit RTX 3080 OC

**Additional context**
I haven't tested this in Windows 10 or any other Windows 11 versions, but I can reproduce it consistently on dev channel.

**Comment by arielj** [comment]
I tried doing some research here, I don't seem to find any issue with white spaces. If I create a folder with white spaces in the name it works fine. I had to actually breack the code for it to create the incorrect `c:/Program` folder named in the description. Maybe something changed? Tried with windows 10 inside a virtual machine.

What I DID notice is that writing to `Program Files` is not allowed by default, I get an access denied error when legendary tries to create the game directory. If I run heroic as an administrator I can pick the Program Files folder and the game is installed and can be played correctly (even with my system in spanish were the `Program Files` folder appears as `Archivos de programa`, so it seems work fine with spaces and with multilingual support).

Maybe there could be a warning to the user if they selected a directory that is not writable?
